### DISCUSSION TITLE
How to locate the gene region

### DISCUSSION THREAD

**Discussion by dahaigui** [GMOD/jbrowse-components#1528] [General]
How to locate the gene region I want to see by the name of the genome, the name of the chromosome and the position，the old version can be accessed through url routing, just like this http://mysite/jbrowse/?data=Brapa15&loc=A03%3A25423663..25425520&tracks=DNA%2CBrapa15&highlight=

**Comment by cmdcolin** [comment]
We don't have anything yet for this, but we are discussing adding this here

https://github.com/GMOD/jbrowse-components/issues/1464

It has just been tricky to get because jbrowse 2 is so highly general but we plan on making this sometime soon

**Comment by dahaigui** [comment]
Thank you very much!

The framework I use is Vue. I hope you can develop a set of code blocks based on Vue. In this way, we can also develop some plug-ins for uploading and communication. If this jbrowse2 can be developed as a component of Vue, it is undoubtedly very good.


Looking forward to your reply!


| |
陈海旭
|
|
dahaiguilalala@163.com
|
签名由网易邮箱大师定制
On 12/7/2020 12:32，Colin Diesh<notifications@github.com> wrote：

We don't have anything yet for this, but we are discussing adding this here

#1464

It has just been tricky to get because jbrowse 2 is so highly general but we plan on making this sometime soon
